### PR TITLE
PrimeNumberNaive

### PR DESCRIPTION
**Pull request by lukaszkutylowski**

### CODE REVIEW

**Review by MatDls — CHANGES_REQUESTED** [review]
Pleae, take a look at comment on `Fibonacci`

File: PrimeNumberNaive/FunctionNaive.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	private double copyResult;
+	private double floorDoubleResult;
+	private int floorResult;
+
+	public void setFunction(double num) {
+		number = num;
+	}
+
+	public void getFunction() {
+	while(result <= (number - 1)) {
```
Comment (MatDls) [inline PrimeNumberNaive/FunctionNaive.java]: why '-1'?

File: PrimeNumberNaive/FunctionNaive.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		result = number / startValue;
+		copyResult = result;
+		floorDoubleResult = Math.floor(copyResult);
+		floorResult = (int) floorDoubleResult;
```
Comment (MatDls) [inline PrimeNumberNaive/FunctionNaive.java]: maybe inline floor() and casting?

File: PrimeNumberNaive/PrimeNumberNaive.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+program calculate of prime number (2...n) when 'n' is input from user
+*/
+
+import java.util.Scanner;
+
+public class PrimeNumberNaive {
+	
+	public static void main(String[] args) {
+	
+	System.out.print("please enter of 'n' number of range (2...n): ");
```
Comment (MatDls) [inline PrimeNumberNaive/PrimeNumberNaive.java]: will work with 1e+20? if not, than what's the largest number that there is avilble?

File: PrimeNumberNaive/PrimeNumberNaive.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	Scanner input = new Scanner(System.in);
+	
+	int nInt = input.nextInt();
+	double n = (double) nInt;
+	
+	FunctionNaive sendN = new FunctionNaive();
+	sendN.setFunction(n);
+	sendN.getFunction();
+}
```
Comment (MatDls) [inline PrimeNumberNaive/PrimeNumberNaive.java]: indentations are not consistent

**Review by MatDls — CHANGES_REQUESTED** [review]

File: Fibonacci.java [hunk 5]
```diff
@@ -1,25 +0,0 @@
...
-	int intUserInput = input.nextInt();
-	int serieArrayNumber = intUserInput - 1;
-	
-	System.out.print("value of " + intUserInput + " fibonacci serie: ");
-	System.out.println(fibonacciSeries[serieArrayNumber]);
-	
-	System.out.print("value of " + intUserInput + " fibonacci series: ");
-	System.out.println(fibonacciSeries[intUserInput]);
-	}
-}
```
[the remark this answers is not in the document]
Comment (lukaszkutylowski) [reply Fibonacci.java]: I think Fibonacci.java file should be in fibonacci brach only. If I copied this file (maybe my mistake, I just learning handling a git repository) I think I should deleted this for cleanup. This situation results from create new branch from level past branch (not master).

File: PrimeNumberNaive/FunctionNaive.java (hunk 1 above)
Comment (lukaszkutylowski) [reply PrimeNumberNaive/FunctionNaive.java]: It's result form Prime Number properties. Prime Number divide by 1 and divide by itself (n), that;s why we divide in range 2..n-1

File: PrimeNumberNaive/PrimeNumberNaive.java (hunk 3 above)
Comment (lukaszkutylowski) [reply PrimeNumberNaive/PrimeNumberNaive.java]: it works with 1e+9 max. I will change on long type (9e+18)

**Pull request merged**